docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to Phonebook
+
+## What should I know before I get started?
+Before you start contributing be sure that you have already taken a look at our [Documentation](https://t-systems-mms.github.io/phonebook/). There you can find all things you need to know before work with the project.
+
+### Mandatory stuff
+To provide good code quality we are using prettier to format all files the same way. Just make sure that you are using the Plugin [Prettier](https://prettier.io/) in your IDE. You can find our settings [here](https://prettier.io/). 
+
+## How can I contribute?
+### Reporting Bugs
+You can contribute by reporting bugs. For that there are some rules:
+- Before submitting please check that the same issue was not already submitted.
+- Use the [Bug-Report-Template](https://github.com/T-Systems-MMS/phonebook/issues/new?assignees=&labels=bug&template=bug_report.md&title=) and answer the questions.
+
```

Comment:
Maybe we could add sth like this: If you already can imagine how your changes could look like, append a mockup.

Reply:
In the template old, not In the docu